specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,451 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-01-01",
+    "title": "Marketplace RP Service"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.Marketplace/PrivateStores/{PrivateStoreId}/Offers": {
+      "parameters": [
```

Comment:
>"p [](start = 6, length = 2)

Add..
"tags": [
          "PrivateStores"
        ],

Same for other apis.. usually it is resource type.. but it is primarily for grouping the apis when viewing in a swagger UI
[https://swagger.io/docs/specification/grouping-operations-with-tags/](https://swagger.io/docs/specification/grouping-operations-with-tags/)

Reply:
this needs to be added to each of the operations (get, put, delete etc..)

---
In reply to: [368369566](https://github.com/Azure/azure-rest-api-specs/pull/8191#discussion_r368369566) [](ancestors = 368369566)